arches_component_lab/src/arches_component_lab/widgets/utils.ts

```diff
@@ -67,3 +67,64 @@ export function convertISO8601DatetimeFormatToPrimevueDatetimeFormat(
 
     return { dateFormat: processedFormat, shouldShowTime: containsTime };
 }
+
+export function getUpdatedComponentPath(
+    deprecatedComponentPath: string,
+): string {
+    const deprecatedComponentToUpdatedComponentPathMap: Record<string, string> =
+        {
+            "views/components/widgets/text":
+                "arches_component_lab/widgets/NonLocalizedStringWidget/NonLocalizedStringWidget",
+            "views/components/widgets/concept-select":
+                "arches_component_lab/widgets/ConceptSelectWidget/ConceptSelectWidget",
+            "views/components/widgets/datepicker":
+                "arches_component_lab/widgets/DatePickerWidget/DatePickerWidget",
```

Comment:
we should either rename the component to "DataPickerWidget" from "DateWidget" or fix this here.  It might make the most sense to follow current naming convention, in which case we should rename the vue components.